build.gradle

```diff
@@ -3,79 +3,56 @@ plugins {
     id 'jacoco'
     id 'checkstyle'
     id 'pmd'
-    id 'nebula.lint' version '16.8.0'
-    id 'de.fuerstenau.buildconfig' version '1.1.8'
+    id 'nebula.lint' version '17.2.0'
+    id "com.github.gmazzo.buildconfig" version "3.0.3"
     id 'idea'
     id 'maven-publish'
     id 'signing'
     id 'io.github.gradle-nexus.publish-plugin' version '1.1.0'
 }
 
-apply plugin: 'de.fuerstenau.buildconfig'
+apply plugin: 'com.github.gmazzo.buildconfig'
 apply plugin: 'idea'
 apply plugin: 'maven-publish'
 
-String versionNumber = '0.13.2'
-
 group 'com.github.bibsysdev'
-version versionNumber
+version '0.13.3'
+
+wrapper {
```

Comment:
maybe easier monitoring of the gradle wrapper? I can remove it if it is probematic